File: tests/human/x/pas/break_continue.pas
program BreakContinue;
var
  numbers: array[1..9] of integer = (1,2,3,4,5,6,7,8,9);
  i: integer;
begin
  for i := 1 to 9 do
  begin
    if numbers[i] mod 2 = 0 then
      continue;
    if numbers[i] > 7 then
      break;
    Writeln('odd number: ', numbers[i]);
  end;
end.

File: tests/machine/x/fortran/break_continue.f90
program break_continue
  implicit none
  integer :: numbers(9) = (/1,2,3,4,5,6,7,8,9/)
  integer :: i
  do i = 1, size(numbers)
    if (mod(numbers(i),2) == 0) cycle
    if (numbers(i) > 7) exit
    print *, 'odd number:', numbers(i)
  end do
end program break_continue
